TruckMuncher/LoginViewController.swift

```diff
@@ -102,9 +103,13 @@ class LoginViewController: UIViewController, FBLoginViewDelegate {
     }
     
     func successfullyLoggedInAsTruck() {
-        self.dismissViewControllerAnimated(true, completion: { () -> Void in
-            NSNotificationCenter.defaultCenter().postNotificationName("loggedInNotification", object: self, userInfo: nil)
-        })
+        mgr.getFullMenus(atLatitude: 0, longitude: 0, includeAvailability: true, success: { (response) -> () in
+            self.dismissViewControllerAnimated(true, completion: { () -> Void in
+                NSNotificationCenter.defaultCenter().postNotificationName("loggedInNotification", object: self, userInfo: nil)
+            })
+        }) { (error) -> () in
+            println("error fetching full menus \(error)")
```

Comment:
better yet, just put this back to the way it was before. I put this in here and its not needed anymore. This getFullMenus method is called in the MapViewController instead. This method should just be the dismissViewController part